doc/hardware/porting/board_porting.rst

```diff
@@ -310,27 +438,43 @@ application for it.
 Setting Kconfig configuration values is documented in detail in
 :ref:`setting_configuration_values`.
 
-There are three mandatory Kconfig files in the board directory for a board
-named ``plank``:
+There are one mandatory Kconfig file in the board directory, and several
```

Comment:
which one is mandatory :)?

Reply:
that is described under: `Create your board directory`.